app/utils/EitherSupport.scala

```diff
@@ -0,0 +1,31 @@
+package utils
+
+object EitherSupport {
```

Comment:
If we borrow code from other projects, can we also borrow its tests?

Reply:
The tests from `sbr-control-api` have now been added!